package.json

```diff
@@ -60,7 +60,7 @@
     "supertest": "^0.15.0"
   },
   "scripts": {
-    "prepublish": "./node_modules/grunt-cli/bin/grunt build",
-    "test": "./node_modules/grunt-cli/bin/grunt test"
```

Comment:
Does this automatically run against the local grunt in node_modules? Or does it require grunt to be installed globally?